docs/FAQ.md

```diff
@@ -33,6 +33,41 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### What are elided revisions in the output of `jj log`? How can I display them?
+
+"Elided revisions" appears in the log when one revision descends from another,
+both are in the revset, but the revisions connecting them are _not_ in the
+revset.
+
+For example, suppose you log the revset `tyl|mus` which contains exactly two
+revisions:
+
+```sh
+$  jj log -r 'tyl|mus'
+○  musnqzvt me@example.com 1 minute ago 9a09f8a5
+│  Revision C
+~  (elided revisions)
+○  tylynnzk me@example.com 1 minute ago f26967c8
+│  Revision A
+```
+
+Only the two revisions in the revset are displayed. "Elided revisions" is shown
+to indicate that `musnqzvt` descends from `tylynnzk`, but the nodes connecting
+them are not in the revset.
```

Comment:
Nit: I think it also is shown when the distance between is to large in cases where 1000 commits are between both revisions.

Reply:
Are you sure that's true? In the microsoft/winget-pkgs repository, I have 170,000 commits, but "elided" doesn't appear in the output at all .

```powershell
PS> jj log -T '1 ++ "\n"' -r 'all()' | measure -Line

 Lines Words Characters Property
 ----- ----- ---------- --------
170887

PS> jj log -T '1 ++ "\n"' -r 'all()' | sls elided
PS> jj log -T '1 ++ "\n"' -r 'root() | master' | sls elided

~  (elided revisions)

PS>
```

Reply:
Sorry, I didn't remember it correctly as they were introduced for it because the default log revset contains a variaton of `root() | @`. And you cannot see it when using `all()` as it disables log elision by forcing all nodes to be rendered anyway.